diff -up pcmanfm-qt-0.15.0/pcmanfm/settings.cpp.2~ pcmanfm-qt-0.15.0/pcmanfm/settings.cpp
--- pcmanfm-qt-0.15.0/pcmanfm/settings.cpp.2~	2020-05-13 18:36:15.848700556 +0200
+++ pcmanfm-qt-0.15.0/pcmanfm/settings.cpp	2020-05-13 18:37:20.838629179 +0200
@@ -230,8 +230,8 @@ bool Settings::loadFile(QString filePath
     settings.endGroup();
 
     settings.beginGroup(QStringLiteral("Desktop"));
-    wallpaperMode_ = wallpaperModeFromString(settings.value(QStringLiteral("WallpaperMode")).toString());
-    wallpaper_ = settings.value(QStringLiteral("Wallpaper")).toString();
+    wallpaperMode_ = wallpaperModeFromString(settings.value(QStringLiteral("WallpaperMode"), QStringLiteral("fit")).toString());
+    wallpaper_ = settings.value(QStringLiteral("Wallpaper"), QStringLiteral("/usr/share/wallpapers/mdk/default.png")).toString();
     wallpaperDialogSize_ = settings.value(QStringLiteral("WallpaperDialogSize"), QSize(700, 500)).toSize();
     wallpaperDialogSplitterPos_ = settings.value(QStringLiteral("WallpaperDialogSplitterPos"), 200).toInt();
     lastSlide_ = settings.value(QStringLiteral("LastSlide")).toString();
