diff -up pcmanfm-qt-0.15.0/config/pcmanfm-qt/lxqt/settings.conf.in.1~ pcmanfm-qt-0.15.0/config/pcmanfm-qt/lxqt/settings.conf.in
--- pcmanfm-qt-0.15.0/config/pcmanfm-qt/lxqt/settings.conf.in.1~	2020-04-23 20:13:46.000000000 +0200
+++ pcmanfm-qt-0.15.0/config/pcmanfm-qt/lxqt/settings.conf.in	2020-05-13 18:34:53.771894491 +0200
@@ -1,21 +1,26 @@
 [System]
+FallbackIconThemeName=breeze
 IconThemeName=elementary
 SuCommand=lxsudo dbus-run-session -- %s
-TerminalCommand=
-Archiver=file-roller
+TemplateRunApp=false
+Archiver=ark
 SIUnit=false
-TerminalDirCommand=xterm
-TerminalExecCommand=xterm -e %s
+Terminal=qterminal
+TerminalCommand=qterminal
+TerminalDirCommand=qterminal
+TerminalExecCommand=qterminal -e %s
 
 [Behavior]
 BookmarkOpenMethod=0
 UseTrash=true
 SingleClick=false
 ConfirmDelete=true
+ConfirmTrash=true
+NoUsbTrash=true
 
 [Desktop]
 WallpaperMode=stretch
-Wallpaper=@LXQT_SHARE_DIR@/themes/frost/lxqt-origami-light.png
+Wallpaper=/usr/share/mdk/backgrounds/default.png
 BgColor=#000000
 FgColor=#ffffff
 ShadowColor=#000000
@@ -28,6 +33,7 @@ Font="Sans Serif,10,-1,5,50,0,0,0,0,0"
 MountOnStartup=true
 MountRemovable=true
 AutoRun=true
+CloseOnUnmount=true
 
 [FolderView]
 Mode=icon
