--- pcmanfm-qt-20140803/libfm-qt/CMakeLists.txt.0000~	2014-08-03 19:50:22.961679560 +0200
+++ pcmanfm-qt-20140803/libfm-qt/CMakeLists.txt	2014-08-03 19:50:53.561891554 +0200
@@ -111,7 +111,7 @@ target_link_libraries(${LIBRARY_NAME}
 # set libtool soname
 set_target_properties(
   ${LIBRARY_NAME}
-  PROPERTIES SOVERSION "0.0.0"
+  PROPERTIES VERSION "0.0.0" SOVERSION "0"
 )
 
 # install include header files (FIXME: can we make this cleaner? should dir name be versioned?)
