diff -up pcmanfm-qt-1.2.0/config/pcmanfm-qt/lxqt/settings.conf.in.1~ pcmanfm-qt-1.2.0/config/pcmanfm-qt/lxqt/settings.conf.in
--- pcmanfm-qt-1.2.0/config/pcmanfm-qt/lxqt/settings.conf.in.1~	2022-11-05 09:07:39.000000000 +0100
+++ pcmanfm-qt-1.2.0/config/pcmanfm-qt/lxqt/settings.conf.in	2022-11-05 19:39:20.661558071 +0100
@@ -15,7 +15,7 @@ ConfirmDelete=true
 [Desktop]
 DesktopShortcuts=Home, Trash, Computer, Network
 WallpaperMode=stretch
-Wallpaper=@LXQT_SHARE_DIR@/wallpapers/waves-logo.png
+Wallpaper=/usr/share/mdk/backgrounds/default.png
 BgColor=#000000
 FgColor=#ffffff
 ShadowColor=#000000
@@ -28,6 +28,7 @@ Font="Sans Serif,12,-1,5,50,0,0,0,0,0"
 MountOnStartup=true
 MountRemovable=true
 AutoRun=true
+CloseOnUnmount=true
 
 [FolderView]
 Mode=icon
