diff -up pcmanfm-qt-2.4.0/pcmanfm/settings.cpp.2~ pcmanfm-qt-2.4.0/pcmanfm/settings.cpp
--- pcmanfm-qt-2.4.0/pcmanfm/settings.cpp.2~	2026-04-20 00:00:00.000000000 +0000
+++ pcmanfm-qt-2.4.0/pcmanfm/settings.cpp	2026-04-20 00:00:00.000000000 +0000
@@ -244,8 +244,8 @@ bool Settings::loadFile(QString filePath
     settings.beginGroup(QStringLiteral("Desktop"));
     screenNames_ = settings.value(QStringLiteral("ScreenNames")).toStringList();
     screenNames_.removeDuplicates();
-    wallpaperMode_ = wallpaperModeFromString(settings.value(QStringLiteral("WallpaperMode")).toString());
-    wallpaper_ = settings.value(QStringLiteral("Wallpaper")).toString();
+    wallpaperMode_ = wallpaperModeFromString(settings.value(QStringLiteral("WallpaperMode"), QStringLiteral("fit")).toString());
+    wallpaper_ = settings.value(QStringLiteral("Wallpaper"), QStringLiteral("/usr/share/wallpapers/mdk/default.png")).toString();
     wallpaperDialogSize_ = settings.value(QStringLiteral("WallpaperDialogSize"), QSize(700, 500)).toSize();
     wallpaperDialogSplitterPos_ = settings.value(QStringLiteral("WallpaperDialogSplitterPos"), 200).toInt();
     lastSlide_ = settings.value(QStringLiteral("LastSlide")).toString();
