diff -uraN pcmanfm-qt-0.15.1/config/pcmanfm-qt/lxqt/settings.conf.in omv-pcmanfm-qt-0.15.1/config/pcmanfm-qt/lxqt/settings.conf.in
--- pcmanfm-qt-0.15.1/config/pcmanfm-qt/lxqt/settings.conf.in	2020-05-17 07:31:18.000000000 +0200
+++ omv-pcmanfm-qt-0.15.1/config/pcmanfm-qt/lxqt/settings.conf.in	2020-05-22 21:46:09.572278860 +0200
@@ -1,21 +1,26 @@
 [System]
+FallbackIconThemeName=breeze
 IconThemeName=elementary
 SuCommand=lxsudo dbus-run-session -- %s
-TerminalCommand=
+TemplateRunApp=false
 Archiver=lxqt-archiver
 SIUnit=false
-TerminalDirCommand=xterm
-TerminalExecCommand=xterm -e %s
+Terminal=qterminal
+TerminalCommand=qterminal
+TerminalDirCommand=qterminal
+TerminalExecCommand=qterminal -e %s
 
 [Behavior]
 BookmarkOpenMethod=0
 UseTrash=true
 SingleClick=false
 ConfirmDelete=true
+ConfirmTrash=true
+NoUsbTrash=true
 
 [Desktop]
 WallpaperMode=stretch
-Wallpaper=@LXQT_SHARE_DIR@/themes/frost/lxqt-origami-light.png
+Wallpaper=/usr/share/mdk/backgrounds/default.png
 BgColor=#000000
 FgColor=#ffffff
 ShadowColor=#000000
@@ -28,6 +33,7 @@
 MountOnStartup=true
 MountRemovable=true
 AutoRun=true
+CloseOnUnmount=true
 
 [FolderView]
 Mode=icon
