--- pcmanfm-qt-0.10.0/CMakeLists.txt.omv~	2016-05-10 12:17:15.467550500 +0200
+++ pcmanfm-qt-0.10.0/CMakeLists.txt	2016-05-10 12:17:22.292577282 +0200
@@ -55,7 +55,6 @@ endif()
 set(CMAKE_AUTOMOC TRUE)
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
-add_subdirectory(libfm-qt)
 add_subdirectory(pcmanfm)
 
 # manpage for pcmanfm-qt
--- pcmanfm-qt-0.10.0/pcmanfm/CMakeLists.txt.omv~	2016-05-10 12:18:36.169874697 +0200
+++ pcmanfm-qt-0.10.0/pcmanfm/CMakeLists.txt	2016-05-10 12:21:12.807795739 +0200
@@ -1,14 +1,16 @@
-set(LIBFM_LIBRARY "fm-qt5")
+set(LIBFM_LIBRARY "fm-qt")
 set(QTX_INCLUDE_DIRS "")
 set(QTX_LIBRARIES Qt5::Widgets Qt5::DBus Qt5::X11Extras)
 
+add_definitions(-std=gnu++14)
+
 include_directories(
     ${QTX_INCLUDE_DIRS}
     ${LIBFM_INCLUDE_DIRS}
     ${LIBMENUCACHE_INCLUDE_DIRS}
     ${SYSTEM_LIBS_INCLUDE_DIRS}
     "${CMAKE_CURRENT_BINARY_DIR}"
-    "${PROJECT_SOURCE_DIR}/libfm-qt"
+    "/usr/include/libfm-qt"
 )
 
 link_directories(
